chart/values.yaml

```
@@ -1542,6 +1542,8 @@ pgbouncer:
   # annotations to be added to the PgBouncer deployment
   annotations: {}
 
+  extraAnnotations: {}
```

```suggestion
  podAnnotations: {}
```

Let's use podAnnotations instead for consistency across components.